JsonSchema.Tests/FormatTests.cs

```diff
@@ -52,6 +52,19 @@ public void DateTime_Pass(string dateString)
 		Assert.True(result.IsValid);
 	}
 
+	[Test]
+	public void DateTime_MissingTimeOffset_Fail()
+	{
+		JsonSchema schema = new JsonSchemaBuilder()
+			.Format(Formats.DateTime);
+
+		var value = JsonNode.Parse("\"2023-04-28T21:51:26\"");
+
+		var result = schema.Evaluate(value, new EvaluationOptions { RequireFormatValidation = true });
+
+		Assert.False(result.IsValid);
```

Comment:
Instead of a simple Assert here, please use the `.IsInvalid()` extension on the result.  This prints the schema and make it easier to debug if something goes wrong.